src/services/auth/index.tsx

```diff
@@ -107,6 +107,11 @@ export const liveRequestOTP = async (
   const payload = { code, phone: mobileNumber };
   Sentry.captureMessage("REQUESTING_OTP", {
     level: "info",
+    tags: {
+      mobileNumber,
+      code,
+      endpoint,
+    },
```

Comment:
tags should appear [here](https://gds-sally.sentry.io/issues/5481584761/tags/?project=5199738), and helps with indexing/searching